### ISSUE TITLE
Bug: UI Screen is flooded by notifications 

### ISSUE THREAD

**Issue by vmarpadge** [topcoder-platform/beta-testing-forums#103]
The Notifications are flooding the UI screen 

Because:  we cannot see the screen to navigate.

Can be improved: By adding clear all notifications button and lessen the size of notification
![image](https://user-images.githubusercontent.com/22873052/109331616-10a17e00-7883-11eb-8342-db1dbc79179d.png)

**Comment by giosal** [comment]
Definitely, although my screen doesn't get flooded completely (full height), but there should be a limit of how much height these notifications can use, like 30-50%, for example
